css/mobile.scss

```
@@ -1,3 +1,5 @@
+$breakpoint-mobile: 1024px;
+
 @media only screen and (max-width: $breakpoint_mobile) {
```

This appears to be the only usage of `$breakpoint-mobile` and apparently everything else in this file ist already plain css. Why not rename the file to `mobile.css` and ditch the variable?
```suggestion
@media only screen and (max-width: 1024px) {
```

---

Please also remove the added variable in L1 and rename the file (I couldn't do it in the suggested change).